### DISCUSSION TITLE
esphome: How can I add user and password for the dashboard?

### DISCUSSION THREAD

**Discussion by oohiggsoo** [tteck/Proxmox#2003] [Ideas]
Hello, I'm trying to set a user + password for the dashboard, unfortunately it doesn't work with the command "esphome dashboard --username USER --password MYPASSWORD configuration".
Greetings

**Comment by tteck** [comment]
`nano /etc/systemd/system/esphomeDashboard.service`

change `ExecStart=/usr/local/bin/esphome dashboard /root/config/` to `ExecStart= /usr/local/bin/esphome dashboard --username USER --password MYPASSWORD /root/config/`
